services/hexathons/src/routes/visit.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-underscore-dangle */
+import { apiCall, asyncHandler, BadRequestError, checkAbility } from "@api/common";
+import { Service } from "@api/config";
+import express from "express";
+import { FilterQuery } from "mongoose";
+
+import { VisitModel, Visit } from "../models/visit";
+
+export const visitRouter = express.Router();
+
+// get visit
+visitRouter.route("/").get(
+  checkAbility("read", "Visit"),
+  asyncHandler(async (req, res) => {
+    const company = await apiCall(
+      Service.USERS,
+      { method: "GET", url: `/companies/employees/${req.user?.uid}` },
+      req
+    );
+    const filter: FilterQuery<Visit> = {};
```

Comment:
You should only be able to read the visits if you're associated with a company. So if there's no company, you'll need to throw an error.